tests/testthat/test-connect_to_etn.R

```diff
@@ -1,3 +1,5 @@
+skip_if_not_localdb()
```

Comment:
Skips are typically done within `test_that()` functions.

- Are you doing it at file level to avoid too much repetition?
- Does that indeed cancel any following tests?

Reply:
Indeed to avoid repetition, and it does work otherwise R CMD CHECK would fail on this branch :sweat_smile: